node-feature-discovery-0.15.yaml

```diff
@@ -0,0 +1,94 @@
+package:
+  name: node-feature-discovery-0.15
+  version: 0.15.4
+  epoch: 0
+  description: Node feature discovery for Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - protoc
+    provides:
+      - node-feature-discovery=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - protoc
+  environment:
+    GRPC_GO_LOG_SEVERITY_LEVEL: "INFO"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-sigs/node-feature-discovery
+      tag: v${{package.version}}
+      expected-commit: 08c731f67fddff47b13aad83fafc2d4e745a72cd
+
+  - name: copy conf
+    runs: |
+      mkdir -p ${{targets.contextdir}}/etc/kubernetes/node-feature-discovery/
+      cp ./deployment/components/worker-config/nfd-worker.conf.example ${{targets.contextdir}}/etc/kubernetes/node-feature-discovery/nfd-worker.conf
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd/kubectl-nfd
+      output: kubectl-nfd
+      tags: osusergo,netgo
+      ldflags: "-X sigs.k8s.io/node-feature-discovery/pkg/version.version=${{package.version}} -X sigs.k8s.io/node-feature-discovery/pkg/utils/hostpath.pathPrefix=/host-"
+      go-package: go
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd/nfd-gc
+      output: nfd-gc
+      tags: osusergo,netgo
+      ldflags: "-X sigs.k8s.io/node-feature-discovery/pkg/version.version=${{package.version}} -X sigs.k8s.io/node-feature-discovery/pkg/utils/hostpath.pathPrefix=/host-"
+      go-package: go
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd/nfd-master
+      output: nfd-master
+      tags: osusergo,netgo
+      ldflags: "-X sigs.k8s.io/node-feature-discovery/pkg/version.version=${{package.version}} -X sigs.k8s.io/node-feature-discovery/pkg/utils/hostpath.pathPrefix=/host-"
+      go-package: go
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd/nfd-topology-updater
+      output: nfd-topology-updater
+      tags: osusergo,netgo
+      ldflags: "-X sigs.k8s.io/node-feature-discovery/pkg/version.version=${{package.version}} -X sigs.k8s.io/node-feature-discovery/pkg/utils/hostpath.pathPrefix=/host-"
+      go-package: go
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd/nfd-worker
+      output: nfd-worker
+      tags: osusergo,netgo
+      ldflags: "-X sigs.k8s.io/node-feature-discovery/pkg/version.version=${{package.version}} -X sigs.k8s.io/node-feature-discovery/pkg/utils/hostpath.pathPrefix=/host-"
+      go-package: go
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes-sigs/node-feature-discovery
+    strip-prefix: v
+    use-tag: true
+    tag-filter: v0.15.
+
+test:
+  pipeline:
```

Comment:
Is there a functional test we can implement outside of retrieving the binary versions/help info?

Reply:
Unfortunately, no, since they all require a kubernetes cluster to be running with env values. Just checked.